src/Option_Command.php

```diff
@@ -295,28 +295,33 @@ public function list_( $args, $assoc_args ) {
 
 		$where = '';
 		if ( $pattern ) {
-			$where .= $wpdb->prepare( "WHERE `option_name` LIKE %s", $pattern );
+			$where .= $wpdb->prepare( 'WHERE `option_name` LIKE %s', $pattern );
 		}
 
 		if ( $exclude ) {
-			$where .= $wpdb->prepare( " AND `option_name` NOT LIKE %s", $exclude );
+			$where .= $wpdb->prepare( ' AND `option_name` NOT LIKE %s', $exclude );
 		}
 		$where .= $autoload_query . $transients_query;
 
-		$results = $wpdb->get_results( "SELECT `option_name`,`option_value`,`autoload`" . $size_query
-					. " FROM `$wpdb->options` {$where}" );
+		$results = $wpdb->get_results(
+			'SELECT `option_name`,`option_value`,`autoload`' . $size_query
```

Comment:
@schlessera All other commands have been fixed, only this part of the Option command remains, can you please take look. Thanks